### DISCUSSION TITLE
Intego detects Trivy as a backdoor

### DISCUSSION THREAD

**Discussion by sycured** [aquasecurity/trivy#6053] [Q&A]
### Question

Do you know a way to prevent the quarantine of trivy by Intego on macOS?
I'm using brew to keep trivy updated.

![image](https://github.com/aquasecurity/trivy/assets/60801403/4c9a1270-82c5-4fe5-92d5-8281cebac5cc)

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Operating System

macOS Sonoma

### Version

```bash
Version: 0.49.0
```

**Comment by knqyf263** [comment]
Could you contact Intego and ask them to suppress the false positive?
